srcpkgs/roccat-tools/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'roccat-tools'
+pkgname=roccat-tools
+version=5.7.0
+revision=1
+build_style=cmake
+configure_args="
+ -DDEVICES=arvo;isku;iskufx;kiro;kone;koneplus;konepure;konepuremilitary;konepureoptical;konextd;konextdoptical;kovaplus;kova2016;lua;nyth;pyra;ryosmk;ryosmkfx;ryostkl;savu;skeltr;sova;suora;tyon
+ -DWITH_LUA=5.3
+ -DUDEVDIR=/usr/lib/udev/rules.d -DCMAKE_MODULE_PATH=/usr/share/cmake"
+make_dirs="/var/lib/roccat 2770 root roccat"
+hostmakedepends="pkg-config glib-devel"
+makedepends="lua-devel gtk+-devel libgaminggear-devel dbus-glib-devel
+ libgudev-devel"
+short_desc="Roccat mouse and keyboard drivers"
+maintainer="Alexander Gehrke <void@qwertyuiop.de>"
+license="GPL-2.0-only"
+homepage="https://roccat.sourceforge.net"
+distfiles="${SOURCEFORGE_SITE}/roccat/${pkgname}/${pkgname}-${version}.tar.bz2"
+checksum=a6e21315c06e072fcea79c185518c8d2c8f19ed9681633613f26085319bdfd95
+system_groups="roccat"
+nocross="https://travis-ci.org/void-linux/void-packages/jobs/477591350#L1740"
```

Comment:
please provide a instructive message of the problem

`Compiler tries to run compiled code for testing`